Source/RageCommunity.Library/Peds/Models/Animal.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace RageCommunity.Library.Peds.Models
+{
+    public enum Animal
```

Comment:
The AmbientMale enum inherits from Int64. I don't understand why Animal enum does not. Is there a specific reason?

Reply:
Certain enums have to be Int64 because of how large their hash number is.